pages/overview/what-is-jsonschema.md

```diff
@@ -6,36 +6,287 @@ title: What is JSON Schema?
 * **Reference**: This section provides a [glossary]((/learn/glossary), [specification release notes](/specification), and [in-depth reference](/understanding-json-schema) to JSON Schema that you can use to write and understand more complex schemas.
 
 
-## Why JSON Schema?
+## JSON Schema History
 
-JSON Schema can:
+JSON Schema has a rich history that dates back to the [first JSON Schema proposal](https://web.archive.org/web/20071026185150/http://json.com/json-schema-proposal/) submitted by **Kris Zyp** to json.com on October 2nd, 2007. 
 
-* Describe your existing data formats
-* Provide clear documentation that’s both human and machine readable
-* Validate your data, which helps you:
-    * Automate testing
-    * Ensure the quality of client-submitted data
-
-JSON Schema is highly extensible and can be tailored to fit your needs. You can create custom keywords, formats, and validation rules to suit your own requirements. The JSON Schema community has a wealth of tools and resources available across many programming languages to help you create, validate, and integrate your schemas.
+The current version of JSON Schema is [2020-12](../draft/2020-12/release-notes.md), which represents the latest advancements and have expended capabilities as compared with the previous version `draft-04`, `draft-06`, `draft-07`. We encourage everyone to adopt the latest version whenever possible to take advantage of all the advancements and benefits of JSON Schema.
 
+For more information regarding JSON Schema history, you can refer to [this article](https://modern-json-schema.com/what-is-modern-json-schema) by **Henry Andrews**.
 
 ## How it works
 
 Using JSON Schema, you can define rules and constraints that JSON data should adhere to. When your JSON documents adhere to these constraints, it becomes easier to exchange structured data between applications because the data follows a consistent pattern.
 
-You begin by defining the rules in a JSON Schema document using various [keywords](https://json-schema.org/draft/2020-12/json-schema-validation.html#name-validation-keywords-for-any). For example, you can use `type` to specify what types of data are valid for a given key. You can specify either a single type, like `string`, or an array of possible types, like `string`, `number`, or `integer`. You can also enforce formats using regular expressions with the `pattern` keyword. There are countless ways to describe specific data types and formats that your data must follow.
+### What is a JSON Document
+
+Before we get into JSON Schema and how it can help us, let's first understand what exactly is a JSON document.
+
+* A JSON document represents a piece of data that follows the syntax and structure defined by the JSON format. It is a collection of key-value pairs, arrays, and nested objects. 
+* JSON documents are used to store and transfer data between systems and applications.
+
+Taking an example of a JSON document representing a customer order,
```

Comment:
Replace , by : at the end of this sentence "Taking an example of a JSON document representing a customer order"